BUILDING.md

```diff
@@ -148,6 +148,121 @@ an installation tree under `build/install`.
 
 If the build did not succeed, please see the next section.
 
+## Building with `conan`
+
+As of version 0.0.11 there is experimental support for building with
+`conan` using the `conanfile.py`. Currently, the only supported
+configuration is that specified in the `default_options` of the
+`conanfile.py`.
+
+To use `conan`, you'll need to have `python` and `pip` on your system if
+you don't already. For example,
+
+- Debian: `apt-get install python3 python3-pip`
+- MacOS with Homebrew: `brew install python`
+
+and then, optionally in a venv if you prefer,
+
+```shell
+pip install conan
+conan profile detect
+```
+
+The second line detects a default [`conan`
+profile](https://docs.conan.io/2/reference/config_files/profiles.html)
+encapsulating your build environment, which you can modify as needed.
+
+Next note that there are two possible approaches here, which complement
+or supersede some of the other sections in this document. Namely, it is
+possible to
+
+1. use `conan` to build and package the SDK; OR
+2. use `conan` to obtain the [software
+   prerequisites](#software-prerequisites) for the SDK.
+
+Option 1. makes more sense for [building against the
+SDK](#building-against-the-sdk) as you would in module development.
+There we use `conan` to get the SDK's dependencies, and to  build,
+install, and package the SDK, which can then be consumed by declaring it
+as a `conan` dependency.
+
+Option 2. makes more sense for doing locally development *on* the SDK
+while using `conan` to get dependencies instead of your system package
+manager. Note that Option 1 implies a superset of Option 2.
+
+### Creating and consuming the SDK `conan` package.
```

Comment:
This is option 1 right? Can we do `Option 1: Creating and consuming the SDK `conan` package.`